app/electron/main.js

```diff
@@ -170,8 +196,16 @@ app.on('ready', createWindow);
 
 app.on('quit', function () {
   if (serverProcess) {
-    serverProcess.stdin.pause();
-    serverProcess.kill();
+    if (serverProcessQuit) {
+      log.error('server process already not running');
+    } else {
+      intentionalQuit = true;
+      log.info('stopping server process...');
+      serverProcess.stdin.pause();
+      // TODO: should we try and end the process a bit more gracefully?
```

Comment:
We've been using `// @todo: `. Not too important but maybe we keep it consistent so it's easy to find in the future.
Maybe the phrasing could also be styled as a todo instead of a question.

Reply:
Ok, thanks. It's done.